build.gradle

```diff
@@ -7,9 +7,9 @@ plugins {
     id "com.github.hierynomus.license-report" version "0.16.1"
 }
 
-version = '2.0.0'
-String bfversion = "7.1.0"
-String b2rversion = "0.9.1"
+version = '2.0.1'
+String bfversion = "7.2.0"
```

Comment:
Note latest Bio-Formats is 7.3.0; I assume the Bio-Formats version here is meant to match what's in bioformats2raw 0.9.2 (which is 7.2.0).

Reply:
Yup, thanks!